src/libraries/Microsoft.PowerFx.Core/Public/Values/RecordValue.cs

```diff
@@ -322,6 +322,7 @@ public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettin
             sb.Append("}");
         }
 
+        [Obsolete("This will be removed in a future version.")]

```

Comment:
>This will be removed in a future version. [](http://example.com/codeflow?start=19&length=41)

If someone is using it, what should they use instead?